### PR TITLE
feat(theme-chalk): css var for empty & rate

### PR DESCRIPTION
**Pull request by MoConWu**
修改empty和rate的色彩作为变量引用，方便后续的色彩更改！

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/theme-chalk/src/common/var.scss [hunk 1]
```diff
@@ -940,6 +940,10 @@ $rate: map.merge(
     'icon-size': 18px,
     'icon-margin': 6px,
     'icon-color': var(--el-text-color-placeholder),
+    'void-color': #c6d1de,
+    'star-color': #F7BA2A,
+    'disable-void-color': #EFF2F7,
+    'text-color': #1f2d3d,
```
Comment (YunYouJun) [inline packages/theme-chalk/src/common/var.scss:946]: Let's unify it all in lowercase here. Or maybe you can lint theme by `pnpm format`.

**Comment by YunYouJun** [comment]
Well done, can you also describe it in the documentation?

File: packages/theme-chalk/src/common/var.scss (hunk 1 above)
Comment (MoConWu) [reply packages/theme-chalk/src/common/var.scss:946]: Thanks!

**Comment by MoConWu** [comment]
feat(theme-chalk): css var for empty & rate.

修改了feat(theme-chalk): css var中的 empty 和 rate 变量列表，提取色彩到变量，方便后续修改。

**Comment by YunYouJun** [comment]
我的意思是，能否在文档里也描述一下，此次的改动，比如在 `docs/en-US/component/empty.md` 中写上，我们为哪里添加了什么名字的 CSS 变量，因此可以通过 CSS 变量来修改 empty 颜色。（当然文档里是用英文）

**Comment by MoConWu** [comment]
> 我的意思是，能否在文档里也描述一下，此次的改动，比如在 `docs/en-US/component/empty.md` 中写上，我们为哪里添加了什么名字的 CSS 变量，因此可以通过 CSS 变量来修改 empty 颜色。（当然文档里是用英文）

OK, I get it

**Review by YunYouJun — APPROVED** [review]

**Comment by YunYouJun** [comment]
OK, thanks.

By the way, I have an idea that we may not need to generate as many CSS variables for empty.
We can use a color function to automatically calculate it by inline, and the user only needs to give a main color.
Of course, this may have to ask the designer, we can do it later.

File: docs/en-US/component/rate.md [hunk 2]
```diff
@@ -84,6 +84,18 @@ Read-only Rate is for displaying rating score. Half star is supported.
...
 rate/readonly
 
+## CustomStyles
+
+Now you can set custom style for rate component.
+Use `css/scss` language to change the global or local color. We set some global color variables: `--el-rate-void-color`, `--el-rate-star-color`, `--el-rate-disable-void-color`, `--el-rate-text-color`. You can use like: `:root { --el-rate-void-color: red; --el-rate-star-color: blue; }`.
+
+### Default Variables
+
+- --el-rate-void-color:  \#c6d1de
```
Comment (YunYouJun) [inline docs/en-US/component/rate.md:94]: Hey, a space should to be deleted. Only lint pass, i can merge.

**Review by YunYouJun — CHANGES_REQUESTED** [review]

**Comment by MoConWu** [comment]
> OK, thanks.
> 
> By the way, I have an idea that we may not need to generate as many CSS variables for empty. We can use a color function to automatically calculate it by inline, and the user only needs to give a main color. Of course, this may have to ask the designer, we can do it later.

Yes, you are right. I'll try it!

File: docs/en-US/component/rate.md (hunk 2 above)
Comment (MoConWu) [reply docs/en-US/component/rate.md:94]: OK

**Comment by YunYouJun** [comment]
Hey, just run `pnpm format`, then commit it.

For example:

image

**Comment by MoConWu** [comment]
> Hey, just run `pnpm format`, then commit it.
> 
> For example:
> 
> image

OK, Thanks. I modified it directly on GitHub before. I'll push it locally this time

**Review by YunYouJun — APPROVED** [review]

**Pull request merged**